### DISCUSSION TITLE
Does CEF not support datalist tags？

### DISCUSSION THREAD

**Discussion by xacxy** [cefsharp/CefSharp#4078] [General]
cefSharp edition 100.0.230.0
using  WinForms  program   
win10 operating  system   
.net FrameWork 4.5.2

**Comment by amaitland** [comment]
In future please use discussions to ask questions. 

CEF does not support data lists in the alloy runtime which is what CefSharp uses.

https://bitbucket.org/chromiumembedded/cef/issues/906/cef3-add-html5-datalist-support

The chrome runtime referenced in the issue should be usable in CefSharp starting in version 102.

**Reply by amaitland** [reply]
For those interested in using the `Chrome Runtime` see #4123
